.github/workflows/static_code_checks.yaml

```diff
@@ -41,11 +41,5 @@ jobs:
         uses: pypa/gh-action-pip-audit@v1.0.8
         with:
           virtual-environment: .venv/
-          # ignoring security vulnerabilities in cryptography
-          # because flwr 1.7 depends on them. Opening an issue
-          # with flwr. Likely can remove this in flwr 1.8.
           ignore-vulns: |
-            GHSA-3ww4-gg4f-jr7f
-            GHSA-9v9h-cgj8-h64p
-            GHSA-6vqw-3v5j-54x4
             GHSA-cqh9-jfqr-h9jj
```

Comment:
This one can also be removed, it has been withdrawn:

https://github.com/advisories/GHSA-cqh9-jfqr-h9jj